godot-macros/src/lib.rs

```diff
@@ -561,11 +572,22 @@ use crate::util::{bail, ident, KvParser};
     alias = "export",
     alias = "tool",
     alias = "rename",
-    alias = "internal"
+    alias = "internal",
+    alias = "icon"
 )]
 #[proc_macro_derive(
     GodotClass,
-    attributes(class, base, hint, var, export, export_group, export_subgroup, init)
+    attributes(
+        class,
+        base,
+        hint,
+        var,
+        export,
+        export_group,
+        export_subgroup,
+        init,
+        icon
```

Comment:
icon is not a stand-alone attribute, but part of `#[class(...)]` attribute, ergo it doesn't belong here.